sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -31,19 +31,19 @@ Tests in the Event Hubs client library are split into two categories:
 
 The Live tests read information from the following environment variables:
 
-`EVENT_HUBS_RESOURCEGROUP`  
+`EVENTHUB_RESOURCE_GROUP`  
```

Comment:
This isn't the correct name for the product;  we should keep "EVENT_HUBS"

Reply:
This will fall into the same bucket as similar discussions.   This needs to be addressed as part of the conversations to fix the repository path.   Resolving, as there's nothing that can be done here.